examples/XX-anomaly-detection.ipynb

```diff
@@ -0,0 +1,1355 @@
+{
```

Comment:
<p>replace width(s) by component(s) :) The width is always fixed and equal to 2 for this time series, as it always has 2 components.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/1518/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>